### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by tank157**
First of two PRs for [the profile page](https://www.pivotaltracker.com/story/show/163699008)

The other PR is [here](https://github.com/votinginfoproject/dasher/pull/5)

The styling is left to be specified later.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -172,13 +176,12 @@
...
       <a href="/" class="branding"><span>VIP Dashboard</span></a>
 
-<!--  #TODO-auth need to confirm user is authenticated-->
-      <!-- <nav class="main-nav" ng-if="user.isAuthenticated"> -->
       <nav class="main-nav" ng-if="user.isAuthenticated">
         <ul style="padding:5px; float: right">
           <li><i class="fi-torso"></i> {{user.userName}}</li>
           <li><a id="pageHeader-feedsLink" href="#/feeds" alert-box error ng-class="{'is-active': pageHeader.section=='feeds'}"><i class="fi-list"></i> Feeds</a></li>
-          <li><a id="pageHeader-sign-out" href="{{logoutUrl}}"><i class="fi-x"></i> Sign Out</a></li>
+					<li><a id="pageHeader-profileLink" href="#/profile" alert-box error ng-class="{'is-active': pageHeader.section=='profile'}"><i class="fi-list"></i> User Profile</a></li>
```
Comment (sduckett) [inline public/index.html]: Does the indentation look a little much, perhaps?

File: public/assets/js/app/services/authService.js [hunk 2]
```diff
@@ -151,7 +151,9 @@ vipApp.factory('$authService', function ($rootScope, $location, $timeout, $http,
             userName: profile["name"],
             email: profile["email"],
             fipsCodes: userToFips(metadata.app_metadata),
-            roles: userToRoles(metadata.app_metadata)
+            roles: userToRoles(metadata.app_metadata),
```
Comment (sduckett) [inline public/assets/js/app/services/authService.js]: funky indentation

File: public/index.html (hunk 1 above)
Comment (tank157) [reply public/index.html]: Yeah, I suspect there's some tab indentation inconsistencies. Let me see what I can do...

**Comment by tank157** [comment]
@sduckett I've fixed my Atom settings and hit up all the files in this PR. Should be a bit nicer going forward.

**Pull request merged**